companion/src/firmwares/opentx/opentxeeprom.cpp

```diff
@@ -128,7 +128,7 @@ inline int SWITCHES_CONFIG_SIZE(Board::Type board, int version)
   return 16;
 }
 
-#define MAX_ROTARY_ENCODERS(board)            (IS_SKY9X(board) ? 1 : 0)
```

Comment:
I wonder if the whole thing can be removed, only the gruvin9x had this at the time, it's old